catalyst-gateway/bin/src/settings/mod.rs

```diff
@@ -437,12 +435,15 @@ impl Settings {
 fn string_to_api_host_names(hosts: &str) -> Vec<String> {
     /// Log an invalid hostname.
     fn invalid_hostname(hostname: &str) -> String {
-        error!("Invalid host name for API: {}", hostname);
+        error!("Invalid host name for API: {hostname}");
```

Comment:
```suggestion
        error!(hostname; "Invalid host name for API!");
```
Don't use string formatting in logs.  Always use labeled data, its much much much easier to process in production.